src/Plugin/Field/FieldFormatter/StrawberryImageFormatter.php

```diff
@@ -215,7 +192,7 @@ public function viewElements(FieldItemListInterface $items, $langcode) {
               //@TODO if no media key to file loading was possible
               // means we have a broken/missing media reference
               // we should inform to logs and continue
-              if ($this->checkAccess($file)) {
+              if (parent::checkAccess($file)) {
```

Comment:
If you are extending a class you can use the $this (i think?) and will back. Parent will always use the parent implementation and maybe someone wants to reimplement locally checkaccess this will skip concrete implementation totally. Does that make sense? Give it a try